### PR TITLE
bump min deploy target to 12

### PR DESCRIPTION
**Pull request by KennyHuRadar**
bumping minimum deploy target and also changing archiving methods to non-deprecated versions.
Uses diffs from @ShiCheng-Lu PR: https://github.com/radarlabs/radar-sdk-ios/pull/394

### CODE REVIEW

File: RadarSDK/RadarReplay.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 #import "Radar.h"
 
-@interface RadarReplay : NSObject <NSCoding>
+@interface RadarReplay : NSObject <NSCoding, NSSecureCoding>
```
Comment (ShiCheng-Lu) [inline RadarSDK/RadarReplay.h:10]: NSSecureCoding inherit from NSCoding, so you don't need both

Comment (KennyHuRadar) [reply RadarSDK/RadarReplay.h:10]: oh yeah lol

**Comment by ShiCheng-Lu** [comment]
we can also remove any `if (@available(iOS ..., *)) {` for `12.0` and lower. There are 2 in VerificationManager and 1 in LocatinoManager

**Review by ShiCheng-Lu — APPROVED** [review]
lgtm

**Comment by ShiCheng-Lu** [comment]
> Let's wait till after the afternoon meeting to ship.
> 
> Also `3.19.0` fails on cocoapods, we'd likely want to bump to `3.19.1`. You can try running the release from this branch before to determine that.

@lmeier were you able to test this https://github.com/radarlabs/radar-sdk-ios/pull/407? can we try to get this in, the changes to the release workflow makes sure the version numbers are ok on pod before release and also checks for lint of both modules before release.

**Pull request merged**